cherrypy/test/test_logging.py

```diff
@@ -127,6 +127,44 @@ def testCustomLogFormat(self):
 
         cherrypy._cplogging.LogManager.access_log_format = original_logformat
 
+    def testUUIDv4ParameterLogFormat(self):
+        '''Test a customized access_log_format string,
+           which is a feature of _cplogging.LogManager.access() '''
+
+        original_logformat = cherrypy._cplogging.LogManager.access_log_format
+        cherrypy._cplogging.LogManager.access_log_format = (
+            '{i}' if six.PY3
+            else
+            '%(i)s'
+        )
+
+        self.markLog()
+        self.getPage('/as_string')
+        self.assertValidUUIDv4()
+
+        cherrypy._cplogging.LogManager.access_log_format = original_logformat
+
+    def testTimezLogFormat(self):
+        '''Test a customized access_log_format string,
```

Comment:
Use double quotes for docstrings and follow PEP257

Reply:
sorry, just copied the original testCustomLogFormat (which has ''' )
I also fixed that